pxc/docker/run-test57

```diff
@@ -22,15 +22,19 @@ docker run --rm \
     export MTR_ARGS='${MTR_ARGS}'
     export MTR_REPEAT='${MTR_REPEAT}'
     export MTR_VAULT_TOKEN='${MTR_VAULT_TOKEN}'
+	export FULL_MTR='${FULL_MTR}'
```

Comment:
Please fix formatting issues: previous lines used 4 spaces for indentation, these now use tabs

Reply:
done